server/public/model/websocket_message.go

```diff
@@ -104,6 +104,11 @@ type WebsocketBroadcast struct {
 	// ReliableClusterSend indicates whether or not the message should
 	// be sent through the cluster using the reliable, TCP backed channel.
 	ReliableClusterSend bool `json:"-"`
+
+	// BroadcastHooks is a slice of hooks IDs used to process events before sending them on individual connections. The
+	// IDs should be understood by the WebSocket code.
+	BroadcastHooks    []string         `json:"-"`
+	BroadcastHookArgs []map[string]any `json:"-"`
```

Comment:
Oh, I thought the cluster send bypassed that somehow since I think I based this on your PR. I'll take a look at this again since I want to make sure the user doesn't get this

Reply:
I didn't get, do you _not_ want to pass this to other nodes in the cluster?

I did that in my PR because that was a function, and you cannot marshal a function, which was the bug.

Reply:
I want it to pass these between nodes in the cluster, but I don't want them to be sent to the client since both of those use the current struct tags. I'll see if there's a way to easily handle those cases differently

Reply:
Ah I see, in that case, you can just use `omitempty` and wipe the fields from the message once the hooks are executed. It will get omitted when json marshalled.